install_scripts/templates/kubernetes/yml-generate.sh

```diff
@@ -342,6 +342,7 @@ spec:
   resources:
     requests:
       storage: "$size"
+  storageClassName: "$STORAGE_CLASS"
```

Comment:
This was removed in a PR a few days ago but it's immutable so the apply failed for existing clusters.